core/commands/cmdenv/env.go

```diff
@@ -40,3 +40,13 @@ func GetConfig(env cmds.Environment) (*config.Config, error) {
 
 	return ctx.GetConfig()
 }
+
+// GetConfigPath extracts the config path from the environment
+func GetConfigPath(env cmds.Environment) (string, error) {
```

Comment:
I would name this `GetConfigRoot`, to be consistent with the variable it is accessing.  I agree it not the best name, but I think it better to be consistent here.

Reply:
@kevina Changed it.